packages/@aws-cdk/aws-pipes-targets-alpha/lib/eventBridge.ts

```diff
@@ -0,0 +1,145 @@
+   *
+   * Minimum: 1
+   * Maximum: 128
+   *
+   * @see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-pipes-pipe-pipetargeteventbridgeeventbusparameters.html#cfn-pipes-pipe-pipetargeteventbridgeeventbusparameters-detailtype
+   * @default none
+   */
+  readonly detailType?: string;
+
+  /**
+   * The URL subdomain of the endpoint.
+   *
+   * For example, if the URL for the endpoint is https://abcde.veo.endpoints.event.amazonaws.com, then `EndpointId` is abcde.veo.
+   *
+   * Pattern: `^[A-Za-z0-9\-]+[\.][A-Za-z0-9\-]+$`
+   * Minimum: 1
+   * Maximum: 50
+   *
+   * @see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-pipes-pipe-pipetargeteventbridgeeventbusparameters.html#cfn-pipes-pipe-pipetargeteventbridgeeventbusparameters-endpointid
+   * @default none
+   */
+  readonly endpointId?: string;
+
+  /**
+   * AWS resources, identified by Amazon Resource Name (ARN), which the event primarily concerns. Any number, including zero, may be present.
+   *
+   * Minimum: 1 | 0
+   * Maximum: 1600 | 10
+   *
+   * @see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-pipes-pipe-pipetargeteventbridgeeventbusparameters.html#cfn-pipes-pipe-pipetargeteventbridgeeventbusparameters-resources
+   * @default none
+   */
+  readonly resources?: string[];
+
+  /**
+   * The source of the event.
+   *
+   * Minimum: 1
+   * Maximum: 256
+   *
+   * @see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-pipes-pipe-pipetargeteventbridgeeventbusparameters.html#cfn-pipes-pipe-pipetargeteventbridgeeventbusparameters-source
+   * @default none
+   */
+  readonly source?: string;
+
+  /**
+   * The time stamp of the event, per RFC3339. If no time stamp is provided, the time stamp of the PutEvents call is used.
+   *
+   * Pattern: (?=[/\.\-_A-Za-z0-9]+)((?!aws\.).*)|(\$(\.[\w/_-]+(\[(\d+|\*)\])*)*)
+   * Minimum: 1
+   * Maximum: 256
+   *
+   * @see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-pipes-pipe-pipetargeteventbridgeeventbusparameters.html#cfn-pipes-pipe-pipetargeteventbridgeeventbusparameters-time
+   * @default none
+   */
+  readonly time?: string;
+}
+
+/**
+ * A EventBridge Pipes target that sends messages to an EventBridge event bus.
+ */
+export class EventBridgeTarget implements ITarget {
+  private eventBus: IEventBus;
+  private eventBridgeParameters?: EventBridgeTargetParameters;
+  public readonly targetArn: string;
+  constructor(eventBus: IEventBus, parameters?: EventBridgeTargetParameters) {
+    this.eventBus = eventBus;
+    this.targetArn = eventBus.eventBusArn;
+    this.eventBridgeParameters = parameters;
+
+    validateDetailType(parameters?.detailType);
+    validateEndpointId(parameters?.endpointId);
+    validateSource(parameters?.source);
+    validateTime(parameters?.time);
```

Comment:
Should cleaner to refactor the functions to take the `parameters` directly, and iterate over the validators

```ts
function validateDetailType({detailType}: EventBridgeTargetParameters)
```

```suggestion
    for (const validate of [validateDetailType, ...]) validate(parameters);
```